diff --git a/browser/installer/package-manifest.in b/browser/installer/package-manifest.in
index 6ebf2f62c477..824b2ac4fd1f 100644
--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -251,6 +251,11 @@
 #endif
 #endif
 
+; Camoufox specific files
+@RESPATH@/defaults/pref/local-settings.js
+@RESPATH@/distribution/policies.json
+@RESPATH@/camoufox.cfg
+
 ; [Default Preferences]
 ; All the pref files must be part of base to prevent migration bugs
 @RESPATH@/browser/@PREF_DIR@/firefox.js
diff --git a/lw/moz.build b/lw/moz.build
index e69de29bb2d1..5459e4ac3653 100644
--- a/lw/moz.build
+++ b/lw/moz.build
@@ -0,0 +1,14 @@
+FINAL_TARGET_FILES += [
+  "camoufox.cfg",
+  "chrome.css",
+  "properties.json"
+]
+
+FINAL_TARGET_FILES.distribution += [
+  "policies.json",
+]
+
+# local-settings does not yet end up being pacakged.
+FINAL_TARGET_FILES.defaults.pref += [
+  "local-settings.js",
+]
diff --git a/moz.build b/moz.build
index a931749b4581..b413fd94117a 100644
--- a/moz.build
+++ b/moz.build
@@ -232,3 +232,5 @@ SPHINX_TREES["content-security"] = "docs/content-security"
 SPHINX_TREES["jsloader"] = "docs/jsloader"
 
 include("build/templates.mozbuild")
+
+DIRS += ["lw"]
